galleries/examples/misc/tickedstroke_demo.py

```diff
@@ -13,7 +13,7 @@
 draws a path with a ticked style.  The spacing, length, and angle of
 ticks can be controlled.
 
-See also the :doc:`contour demo example
+See also the :doc:`lines with a ticked patheffect example
 </gallery/lines_bars_and_markers/lines_with_ticks_demo>`.
```

Comment:
If we want the example title, I think we can just leave out the title:
```suggestion
See also the :doc:`/gallery/lines_bars_and_markers/lines_with_ticks_demo` example.
```